REVIEW.md

```diff
@@ -0,0 +1,219 @@
+      If appropriate, help the contributor understand why tests failed.
+    - Are regression tests needed?
+      If a reproducible bug is being fixed in this PR, it may be needed and reviewers can help the contributor design and implement them.
+      In case the contributor is unable to add the test, the reviewer should add it in a follow-up PR.
+
+ - [ ] Sustainability and maintenance
+    - Is the code easy to read and low on redundancy?
+    - Should variable names be improved for clarity or consistency?
+    - Should comments or docstrings be added?
+    - Should comments be removed as unhelpful or extraneous?
+    - Will the cost of maintaining a new feature be worth its benefits?
+    - Will the new code add any dependencies on other projects?
+
+- [ ] User-facing changes
+    - What use cases might this change influence?
+    - Does the user experience match existing patterns in Spyder?
+      If it does not, why was the choice made?
+    - Is in-interface text (new or modified) appropriately descriptive and free of spelling or grammar issues that would make its meaning unclear?
+
+Review that addresses these questions on a PR will be prefixed with `[required]` so that the PR author(s) are clear that they are blockers.
+PR authors are responsible for addressing these reviews if prompted by the reviewer in order for that PR to be eligible for merging.
+
+### Optional review
+
+The following are themes that may be helpful to cover on a PR, but are optional.
+
+**Note**: The difference between an objective improvement and a subjective preference isn’t always clear.
+Reviewers should recall that code review is primarily about reducing risk in the project.
+When reviewing code, one should aim at preventing situations which may require a bug fix, a deprecation, or a retraction.
+
+- [ ] Improvements
+    - Could the code easily be rewritten to run much more efficiently for relevant settings?
+    - Is the code backwards compatible with previous versions?
+      (Or is a deprecation cycle necessary?)
+    - Is there feedback that would be helpful for solving future issues surrounding this change?
+      Are there non-urgent concerns that the team should be made aware of?
+    - If a bug is fixed, does the PR include a non-regression test?
+    - Are there images that should be added?
+      Are there images that should be resized or optimized in another way?
+
+- [ ] User-facing changes
+    - Does the user interface or other visual elements match existing patterns in Spyder?
+      If it does not, is there a reason?
+
+Review that addresses these questions on a PR will be prefixed with `[optional]` so that the PR author(s) are clear that this feedback is not a blocker.
+Addressing this feedback is considered above-and-beyond, and may be more relevant on some PRs than others.
+PR authors may address these reviews, but are not required to do so in order for the PR to be eligible for merging.
+
+
+## For documentation-focused review
```

Comment:
I've updated the title based on a suggestion you made (that I missed) in a prior document. Does that address this comment?

Reply:
You didn't miss it haha; I just made it too late in the process—my bad.

I actually somewhat revised my thinking on it since then; part of me is favoring something like ``For prose-focused review`` as that more concisely, broadly and appropriately captures the range of potential PRs (and parts thereof) which it applies to. However, on the other hand, that would require some pretty significant rewriting of a fair bit of the text below, versus just adding website a smattering of places where documentation is mentioned (which is all it currently needs now in that case).

Reply:
Given that, to avoid dragging this out further, and that I myself am not sure what if anything we should do further, maybe best to just leave this for now.